enrysrv/cli/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"os"
+
+	"github.com/jessevdk/go-flags"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	appname = "enrysrv"
+)
+
+var (
+	version = "undefined"
+	build   = "undefined"
+	commit  = "undefined"
```

Comment:
Just a nitpick probably and does not block the merge, but would not it be nice to replace this + linker flags in Makefile with `go generate`, what do you think?  Like in https://github.com/a-h/version/blob/master/get_version_linux.sh